drivers/node/node.go

```diff
@@ -152,6 +152,15 @@ type Driver interface {
 
 	// GetZones returns list of zones in which ASG cluster is running
 	GetZones() ([]string, error)
+
+	// PowerOnVM powers VM
+	PowerOnVM(node Node) error
+
+	// PowerOffVM powers VM
+	PowerOffVM(node Node) error
+
+	// RebootVM powers VM
+	RebootVM(node Node) error
```

Comment:
Use the existing RebootNode methods and override them in the vsphere driver implementation

Reply:
Fixed